evalml/tests/component_tests/test_one_hot_encoder.py

```diff
@@ -144,9 +144,9 @@ def test_null_values_in_dataframe():
             "col_4": [2, 0, 1, 3, 0, 1, 2],
```

Comment:
is there a test case where `encoder_error = OneHotEncoder(handle_missing="error")` and it fits on a data frame with a numeric column with a null value?